zebra-scan/src/lib.rs

```diff
@@ -0,0 +1,8 @@
+//! Shielded transaction scanner for the Zcash blockchain.
+
+#![doc(html_favicon_url = "https://zfnd.org/wp-content/uploads/2022/03/zebra-favicon-128.png")]
+#![doc(html_logo_url = "https://zfnd.org/wp-content/uploads/2022/03/zebra-icon.png")]
+#![doc(html_root_url = "https://doc.zebra.zfnd.org/zebra_state")]
```

Comment:
This URL was removed by PR #7886 and the site no longer exists. The crate name is also wrong.

```suggestion
#![doc(html_root_url = "https://docs.rs/zebra_scan")]
```